fifa_draft/utils.py

```diff
@@ -1,18 +1,27 @@
 from django.contrib import messages
-from fifa_draft.models import Group
-import random
+from django.core.handlers.wsgi import WSGIRequest
+from fifa_draft.forms import EditTeamForm, TeamForm
+from fifa_draft.models import Team, Group
+from users.models import Profile
 
 
-def is_unique_name(team):
-    unique_name = True
+def is_unique_name(team: Team) -> bool:
     for team_in_group in team.belongs_group.teams.all():
         if str(team_in_group).lower() == str(team).lower() and team.owner != team_in_group.owner:
-            unique_name = False
-    return unique_name
+            return False
+    return True
```

Comment:
wystarczy:
```
return not str(team_in_group).lower() == str(team).lower() and team.owner != team_in_group.owner:
```

Reply:
niestety, w taki sposób to nie działa, być może to kwestia kodu gdzie ta funkcja jest wykorzystana, przyjrzę się temu

Reply:
Fakt, nie zauważyłem, że ten if jest w pętli, wtedy to by zmieniało logikę, bo return by się wywoływał zawsze